doc/source/whatsnew/v1.3.0.rst

```diff
@@ -595,6 +595,7 @@ I/O
 - Bug in :func:`read_hdf` returning unexpected records when filtering on categorical string columns using ``where`` parameter (:issue:`39189`)
 - Bug in :func:`read_sas` raising ``ValueError`` when ``datetimes`` were null (:issue:`39725`)
 - Bug in :func:`read_excel` dropping empty values from single-column spreadsheets (:issue:`39808`)
+- Bug in :func:`read_excel` failing with ``MultiIndex`` columns followed by two empty rows and no index (:issue:`40442`)
```

Comment:
Should also mention read_csv (at least) I think.

Reply:
Yes, good point.